.github/workflows/tests.yaml

```diff
@@ -19,14 +19,15 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install --no-install-recommends -y graphviz make wget
-          python -m pip install black==20.8b1 mdformat==0.7.5
```

Comment:
I think we should keep the versions `black==20.8b1 mdformat==0.7.5`. Black has different formatting in different versions.

Reply:
Yes, but that is the point, no? Everyone should have the last version or we loose the benefit of the same formatting.

Reply:
I think just, stick with a version and communicate that in the contribution guideline. I don't want `100 files changed`, because black has a new release.